.env.example

```diff
@@ -1,73 +1,73 @@
-# Location and credentials of the postgres database
-DBHOST              = 'postgres'
-DBNAME              = 'genlayer_state'
-DBUSER              = 'postgres'
-DBPASSWORD          = 'postgres'
-DBPORT              = '5432'
+# Postgres Database Configuration
+DBHOST                    = 'postgres'
+DBNAME                    = 'genlayer_state'
+DBUSER                    = 'postgres'
+DBPASSWORD                = 'postgres'
+DBPORT                    = '5432'
 
-# JSON array of initial validators to be created on startup.
-# Example: [{"stake": 100, "provider": "openai", "model": "gpt-4o", "amount": 2}, {"stake": 200, "provider": "anthropic", "model": "claude-3-haiku-20240307", "amount": 1}]
-VALIDATORS_CONFIG_JSON = ''
-
-LOGCONFIG          = 'dev'  # dev/prod
-FLASK_LOG_LEVEL    = 'ERROR'  # DEBUG/INFO/WARNING/ERROR/CRITICAL
+# Logging Configuration
+LOGCONFIG                   = 'dev'     # dev/prod
+FLASK_LOG_LEVEL             = 'ERROR'   # DEBUG/INFO/WARNING/ERROR/CRITICAL
 DISABLE_INFO_LOGS_ENDPOINTS = '["ping", "eth_getTransactionByHash","gen_getContractSchemaForCode","gen_getContractSchema"]'
 
-# JsonRPC server details
-RPCPROTOCOL         = 'http'
-RPCHOST             = 'jsonrpc'
-RPCPORT             = '4000'
-# (the port debugpy is listening on)
-RPCDEBUGPORT        = '4678'
+# JsonRPC Server Configuration
+RPCPROTOCOL             = 'http'
+RPCHOST                 = 'jsonrpc'
+RPCPORT                 = '4000'
+RPCDEBUGPORT            = '4678'      # debugpy listening port
+ENVIRONMENT             = 'dev'       # dev/prod
```

Comment:
where is this being used?

Reply:
I am using docker compose profile for switching the nginx on/off instead of this env var